spotbugs-plugin/src/test/java/com/linkedin/avroutil1/spotbugs/GoodClass.java

```diff
@@ -1,13 +1,21 @@
 package com.linkedin.avroutil1.spotbugs;
 
 import com.linkedin.avroutil1.compatibility.AvroCompatibilityHelper;
+import org.apache.avro.io.BinaryDecoder;
 import org.apache.avro.io.BinaryEncoder;
+import org.apache.avro.io.DecoderFactory;
 
+import java.io.InputStream;
 import java.io.OutputStream;
 
 public class GoodClass {
 
     public void instantiateBinaryEncoder() {
         BinaryEncoder bobTheEncoder = AvroCompatibilityHelper.newBinaryEncoder( (OutputStream) null);
     }
+
+    public void instantiateBinaryDecoder() {
+        BinaryDecoder bobTheDecoder = AvroCompatibilityHelper.newBinaryDecoder( (InputStream) null);
```

Comment:
So... what does spotbugs say if you pass null without casting it? Does it think it's bad?

Reply:
this particular invocation would be ambiguous and wont compile without the cast